qml/Panopticon/Welcome.qml

```diff
@@ -0,0 +1,175 @@
+import QtQuick 2.4
```

Comment:
When i start panopticon via `qtpanopticon tests/data/libfoo.so` the welcome screen is still here until I click on a function, which is a little discombobulating.

Perhaps disassembly should default to first discovered function?